### PR TITLE
Vc/submit profile info

### PR DESCRIPTION
**Pull request by vasuchal**
Form for users to create profiles and sign up

### CODE REVIEW

File: frontend/src/pages/SignupPage.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import { Button, Checkbox, Form } from 'semantic-ui-react'
+import "../styles/Signup.css";
+//import Select from 'react-select'
+import { Dropdown } from 'semantic-ui-react'
+
+const languageOptions = [
```
Comment (alicesf2) [inline frontend/src/pages/SignupPage.js]: replace with real data eventually

File: frontend/src/pages/SignupPage.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  { key: 'Turkish', text: 'Turkish', value: 'Turkish' },
+  { key: 'Vietnamese', text: 'Vietnamese', value: 'Vietnamese' },
+]
+
+
+const options = [
+  { value: 'chocolate', label: 'Chocolate' },
+  { value: 'strawberry', label: 'Strawberry' },
+  { value: 'vanilla', label: 'Vanilla' }
+]
```
Comment (alicesf2) [inline frontend/src/pages/SignupPage.js]: this one as well

File: frontend/src/pages/SignupPage.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    fontSize:"16px",
+    backgroundColor:"#C70039",
+    width:"48%",
+    marginBottom:"15px",
+    textShadow: "2px 2px black",
+    fontWeight:"5",
+    marginRight:"5%",
+    marginLeft:"5%"
+  }
+  const contentAlign = {display:"flex", justifyContent:"center"}
```
Comment (alicesf2) [inline frontend/src/pages/SignupPage.js:9]: move styles to css file

File: frontend/src/styles/Signup.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+.main {
+        display:"flex";
+        width: "60%";
+        margin:"auto";
+        background-color:"black";
+        border:"3px #C70039 outset";
+        border-radius : "15px";
+        justify-content:"center";
+}
```
Comment (alicesf2) [inline frontend/src/styles/Signup.css:77]: remove quotes around values

**Review by alicesf2 — APPROVED** [review]
Great job! The form looks good, just some minor code and file changes.

File: frontend/src/index.js [hunk 5]
```diff
@@ -13,6 +14,7 @@ ReactDOM.render(
     <Router>
       <Switch>
         <Route exact path="/" render={(props) => <Home {...props} />} />
+        <Route exact path="/signuppage" render={(props) => <SignupPage {...props} />} />
```
Comment (coconut750750) [inline frontend/src/index.js]: Just a small nit, but set `path="/signup"`

File: frontend/src/utils/api.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+export async function makePost(name, school, year, age, gender, major, phoneNumber, fbUsername, igUsername, scUsername, email, housingType, numRoommates, shortDesc, longDesc, quizAnswers) {
```
Comment (coconut750750) [inline frontend/src/utils/api.js]: Change this function name to something more descriptive like `submitProfile`

**Review by coconut750750 — DISMISSED** [review]

**Review by alicesf2 — COMMENTED** [review]
Invalid data gets submitted to database upon correct submission, me & brandon will look into this

**Review by alicesf2 — APPROVED** [review]

**Review by coconut750750 — APPROVED** [review]

**Pull request merged**